agir/events/components/eventPage/EventHeader.js

```diff
@@ -103,6 +103,13 @@ const StyledActionButtons = styled.div`
   }
 `;
 
+const StyledButton = styled(Button)`
+  height: 48px;
+  font-size: 16px;
+  font-weight: 500;
+  border-radius: 0.5rem;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```